packages/lab/src/segmented-button-group/SegmentedButtonGroup.css

```diff
@@ -0,0 +1,24 @@
+.saltSegmentedButtonGroup {
+  display: flex;
+  flex-direction: row;
+  background: var(--button-background);
+  gap: var(--salt-size-border);
+}
+
+.saltSegmentedButtonGroup > *:not(:last-child)::after {
+  content: "";
+  width: var(--salt-size-border);
```

Comment:
probably more of a design question but we have some separator variables in use for stack layout, i wonder if they are the same, if we should use them here too (they might be on a deprecating path, idk)

Reply:
from what i can see  --stackLayout-separator-weight is equated to --salt-size-border ? Or is there another variable you're talking about ?